pyaerocom/vert_coords.py

```diff
@@ -112,14 +118,14 @@ def geopotentialheight2altitude(geopotential_height):
 
     Parameters
     ----------
-    geopotential_height
+    geopotential_height : float
         input geopotential height values in m
 
     Returns
     -------
     Computed altitude levels
     """
-
+    raise NotImplementedError("This function is a dummy function")
```

Comment:
can this function be removed?

Reply:
It is being referenced here: https://github.com/metno/pyaerocom/blob/8e09efc1e48105b50bcf385a4f31a0ce53157ce8/pyaerocom/vert_coords.py#L206 but isn't otherwise used. If removing this doesn't break any tests I'll remove it.

Reply:
No tests broke from removing it, so I removed it.